EAD/Session7/spring-demo/src/main/java/com/metacube/training/mailSender/SmtpMailSender.java

```diff
@@ -0,0 +1,14 @@
+package com.metacube.training.mailSender;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Component;
+
+//@Component("smtpmailsender")
+//@Primary
+public class SmtpMailSender implements MailSender {
+	
+	@Override
+	public void sendMail()
+	{
+		System.out.println("MAIL SENT!!! SMTP sender");
```

Comment:
We can use this for print in place of System.out.println
private static final Logger log = LoggerFactory.getLogger(SmtpMailSender.class); 
log.info("Mail send from smtp mail sender");

Reply:
Done